testbed/person_data/data.json

```diff
@@ -0,0 +1,20 @@
+{
+    "People": [
+        {
+            "parents": [
+                {"name": "Henry", "age": 35, "weight": 235, "married": true},
+                {"name": "Martha", "age": 36, "weight": 140, "married": true}
+            ],
+            "children": [
+                {"name": "Steve", "age": 15, "weight": 120, "married": false},
+                {"name": "Alex", "age": 12, "weight": 110, "married": false},
+                {"name": "Sarah", "age": 9, "weight": 70, "married": false}
+            ]
+        },
+        { "name": "Fridge", "age": 42, "weight": 180, "married": false },
+        { "name": "Jane", "age": 57, "weight": 140, "married": true },
+        { "name": "Jane", "age": 58, "weight": 140, "married": true },
```

Comment:
Also, why are there 5 Janes?

Reply:
I updated them to get the order right but it's because we want to compare people that differ in only one attribute for each attribute to ensure that the model checks all attributes for equality. So when the people are loaded and sorted the test just checks the neighboring people for equality.

Reply:
This is probably also confusing to the model? Maybe we should only have the test for it's code do this and not show it this?